src/EditorFeatures/VisualBasic/Outlining/Outliners/AccessorDeclarationOutliner.vb

```diff
@@ -34,7 +17,7 @@ Namespace Microsoft.CodeAnalysis.Editor.VisualBasic.Outlining
                     spans.Add(
                     VisualBasicOutliningHelpers.CreateRegionFromBlock(
                         methodBlock,
-                        GetBannerText(accessorDeclaration),
+                        accessorDeclaration.ConvertToSingleLine().ToString() & " " & Ellipsis,
```

Comment:
Added. I also went through and generally cleaned up the code in the outliners. It's quite a bit smaller now.